src/client/csfle/client_encryption.rs

```diff
@@ -250,25 +296,118 @@ pub struct ClientEncryptionOptions {
     /// Map of KMS provider properties; multiple KMS providers may be specified.
     pub kms_providers: KmsProviders,
     /// TLS options for connecting to KMS providers.  If not given, default options will be used.
-    #[builder(default)]
     pub tls_options: Option<KmsProvidersTlsOptions>,
 }
 
+impl ClientEncryptionOptions {
+    /// Create a new `ClientEncryptionOptions` with only required fields specified.
+    pub fn new(
+        key_vault_client: Client,
+        key_vault_namespace: Namespace,
+        kms_providers: KmsProviders,
+    ) -> Self {
+        Self {
+            key_vault_client,
+            key_vault_namespace,
+            kms_providers,
+            tls_options: None,
+        }
+    }
+}
+
+/// Builder for convenient construction of a new `ClientEncryption`.
+#[derive(Debug)]
+pub struct ClientEncryptionBuilder<const KMS_SET: bool> {
```

Comment:
Is it possible to construct a `ClientEncryptionBuilder<false>` via the public API?